### DISCUSSION TITLE
Feature Request: Sitemap plugin: Allow config to name sitemap file

### DISCUSSION THREAD

**Discussion by jpol0191** [gatsbyjs/gatsby#33018] [Ideas / Feature Requests]
Currently, when using the [gatsby-plugin-sitemap](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-sitemap) you are not able to name the resulting file that contains the sitemap.

I propose that users be allowed to specify the name of the sitemap file in the plugin configs in `gatsby-config.js`.
Perhaps `fileName: "sitemap.xml"` for example. 

It would also be nice if gatsby gave the option to generate a sitemap it self rather than a sitemap index that points to the sitemap.

**Comment by violy** [comment]
The default actual _default_ path is `/sitemap/sitemap-index.xml` where filename `sitemap-index.xml` is hardcoded. The base path is a plugin option, the name not… why ? Many websites have a `/sitemap.xml` path convention. 

Use a plugin option to configure that string is a good idea.
